hwdb.d/60-keyboard.hwdb

```diff
@@ -75,7 +75,13 @@
 #   XKB_FIXED_VARIANT=""
 # Examples of such devices: the Yubikey or other key-code generating
 # devices.
-#
+
+# A device where the scan code to key code mapping in insufficient and
+# requires a special key code to symbol configuration may specify that with:
+#   XKB_MODEL="xkbmodel"
```

Comment:
any specific reason you didn't stick with the existing `XKB_FIXED_*` type of naming? I reckon it may be better as it signals a non-changeable configuration.

also - is there anything that parses this yet? afaict the big drawback of the existing entries for the yubikeys are simply that nothing seems to care about it.

Reply:
> any specific reason you didn't stick with the existing XKB_FIXED_* type of naming?

Not really, no. I'll change it.

> also - is there anything that parses this yet? afaict the big drawback of the existing entries for the yubikeys are simply that nothing seems to care about it.

No but I'm looking at implementing this in gnome shell. It's a good opportunity to wire up the other XKB_FIXED_ properties.